### PR TITLE
Fix merge error & added doc for sensehat new flag

### PR DESCRIPTION
**Pull request by nodinosaur**
**Description:**
Added a new Flag for if the Sensehat is Attached to the R-Pi

Fixed a messed up merge

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#5883

### CODE REVIEW

File: source/_components/sensor.sensehat.markdown [hunk 1]
```diff
@@ -34,56 +35,13 @@ Configuration variables:
   - 'temperature'
   - 'humidity'
   - 'pressure'
+  is_hat_attached (Optional): True|False boolean; True declares the SenseHAT _is_ physically on the Raspberry Pi
```
Comment (fabaff) [inline source/_components/sensor.sensehat.markdown]: Please state what the default is.

File: source/_components/sensor.sensehat.markdown [hunk 2]
```diff
@@ -34,56 +35,13 @@ Configuration variables:
...
+  
 
-### {% linkable_title Installation instruction for the All-In-One installer and HASSbian %}
+#### Customizing the Sense HAT data
```
Comment (fabaff) [inline source/_components/sensor.sensehat.markdown]: This part was already added by https://github.com/home-assistant/home-assistant.github.io/pull/2009. I suggest that you limit your PR to the new Boolean and make it against `next` which is the right branch for new features.

File: source/_components/sensor.sensehat.markdown [hunk 3]
```diff
@@ -26,6 +26,7 @@ sensor:
         - temperature
         - humidity
         - pressure
+    is_hat_attached: True|False
```
Comment (nodinosaur) [inline source/_components/sensor.sensehat.markdown]: sure

File: source/_components/sensor.sensehat.markdown (hunk 1 above)
Comment (nodinosaur) [inline source/_components/sensor.sensehat.markdown]: sure

File: source/_components/sensor.sensehat.markdown (hunk 2 above)
Comment (nodinosaur) [inline source/_components/sensor.sensehat.markdown]: The previous issue has a styling issue when merge occurred. This corrects that on top of that PR. This is the correct version.

**Pull request merged**